search-app/test/cmr/search/test/unit/data/complex_to_simple_converters/has_granules_or_cwic_test.clj

```diff
@@ -0,0 +1,99 @@
+(ns cmr.search.test.unit.data.complex-to-simple-converters.has-granules-or-cwic-test
+  (:require 
+   [clojure.test :refer [deftest is testing]]
+   [cmr.common.cache :as cache]
+   [cmr.common.services.search.query-model :as cqm]
+   [cmr.elastic-utils.search.query-transform :as c2s]
+   [cmr.search.models.query :as query]
+   [cmr.search.services.query-execution.has-granules-or-cwic-results-feature :as has-gran-or-cwic-feature]
+   [cmr.search.services.query-execution.has-granules-results-feature :as has-granules-feature]))
+
+(defn- make-fake-cache
+  "Create a minimal fake cache that satisfies CmrCache protocol."
+  [cache-data]
+  (reify cache/CmrCache
```

Comment:
nice. I had forgotten about `reify`.

Reply:
I didn't even realize we could do this until I saw Leo's code review.  I stole the idea from him.